storage/acl.go

```diff
@@ -172,33 +172,33 @@ func (a *ACLHandle) configureCall(ctx context.Context, call interface{ Header()
 }
 
 func toObjectACLRules(items []*raw.ObjectAccessControl) []ACLRule {
-	var rs []ACLRule
-	for _, item := range items {
-		rs = append(rs, toObjectACLRule(item))
+	rs := make([]ACLRule, len(items))
```

Comment:
This is not really equivalent; if `len(items) == 0`, we get an empty slice rather than a nil. Same elsewhere. This can make a difference in what is sent to the API.